src/app/components/lunch/lunch.component.scss

```diff
@@ -81,74 +71,72 @@
 			border-right: 1px solid rgba($sophisticated-white, 0.5);
 		}
 	}
-
 	.lunch-week {
 		display: flex;
 		justify-content: space-between;
 		width: auto;
 		height: auto;
 		overflow-x: scroll;
-
 		.lunch-day {
 			//width: calc(20% - 0.5rem);
 			flex: 0 0 100%;
 			padding: 0.5rem;
 			color: $sophisticated-white;
-
 			&.today {
 				border: 0.25rem solid #A5001E;
 			}
-
 			.lunch-date {
 				margin-bottom: 1rem;
-
 				h3 {
 					margin-bottom: 0.25rem;
 					font-weight: 300;
 					letter-spacing: 0.03rem;
 				}
-
 				h5 {
 					margin: 0;
 				}
 			}
-
 			.lunch-menu {
 				.no-lunch {
 					padding: 1rem;
 					text-align: center;
 					border-top: 1px solid rgba(white, 0.5);
-
 					p {
 						margin: 0;
 						font-size: 1.25rem;
 						font-weight: 300;
 					}
 				}
-
 				.lunch {
 					.lunch-category {
-						padding-top: 0.5rem;
+						 padding-top: 0.5rem;
 						border-top: 1px solid rgba(white, 0.5);
-
 						h3 {
 							text-align: center;
 							font-weight: 600;
 							letter-spacing: 0.03rem;
-						}
-
-						ul {
+							}
+							ul {
 							margin: 0 0 1rem 0;
 							padding: 0;
 							list-style: none;
-
 							li {
 								font-weight: 300;
-
 								&:before {
-									content: '🍔';
```

Comment:
Could you maybe add a default fallback character in case there's some new category that isn't one of the already specified? You should be able to just keep the `content: '🍔';` for all `li:before` because any of the other specific categories will override the default.